index.yaml

```diff
@@ -217,6 +217,37 @@ entries:
     urls:
     - https://github.com/scalar-labs/helm-charts/releases/download/scalardl-3.2.0/scalardl-3.2.0.tgz
     version: 3.2.0
+  - apiVersion: v2
```

Comment:
Do we manage this file manually?
Do we need a section for each version?
If so, I have some concerns since the file has many redundancies and it seems error-prone to update this file.

Reply:
@feeblefakie 
Yes, you are right.
I think we need to think about how to upgrade the patch versions of the lower major and minor versions operationally.